src/warnet/lnchannel.py

```diff
@@ -1,52 +1,51 @@
 import logging
 
 
+class LNPolicy:
+    def __init__(
+        self,
+        min_htlc: int = 0,
+        max_htlc: int = 0,
+        base_fee_msat: int = 0,
+        fee_rate_milli_msat: int = 0,
+        time_lock_delta: int = 0,
```

Comment:
Not sure if these should have default values at all, but probably doesnt matter because we replace all or nothing

Reply:
Just left default in for time_lock_delta as CLN doesn't give us that.